hortimon/mothership/threshold_alerts/web/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Flask==0.12.2
+twilio==6.10.1
```

Comment:
this file documents the libraries our server depends on, and freezes them to particular values

we can `pip install -r requirements.txt` to have pip set up the environment (note that we run this command in `Dockerfile`)